### ISSUE TITLE
Two suggestions

### ISSUE THREAD

**Issue by lodle** [desura/desura-app#245]
_Issue by **[Yuraconst](https://github.com/Yuraconst)** from Sunday Jun 17, 2012 at 09:11 GMT_
_Originally opened as https://github.com/desura/Desurium/issues/245_

---
1. Add ability to use eSpeak to speak errors as they occur (using eSpeak on Linux and SAPI on Windows) - you don't want your errconsole to blow your fullscreen game.
2. Add Windows, KDE and OSX (when port is ready) gadgets - to launch games using a small on-desktop list.
3. Ship the Linux client with an Oxygen theme and Windows client - with Luna and Aero themes. If you used Chrome not only for webframe but also for UI, it would be possible to integrate Aero into headerbar and change themes w/o restart.
   Basically, 3rd one comes down to an option of fusing the UI style with the OS. An option which is off by default.
